### ISSUE TITLE
Sending a payload with null in it doesn't get rendered

### ISSUE THREAD

**Issue by jasonvarga** [spatie/ray#165]
**Describe the bug**

If I perform a `ray()` call where _any_ argument is `null`, nothing gets rendered in the app.

```php
ray('hello'); // this is rendered
ray('hello', 'there'); // these are both rendered

ray(null); // nothing rendered
ray('hello', null); // nothing rendered
ray('hello', null, 'there'); // nothing rendered
ray(null, 'hello'); // nothing rendered
```

**Versions**
Ray version: 1.3.1

You can use `composer show` to get the version numbers of:
- spatie/ray package version: 1.9.1 (specifically, commit sha 6969977)
- spatie/laravel-ray package version: N/A

PHP version: 7.4.14
Laravel version: N/A

**To Reproduce**
Call `ray()` with a null as the lone argument, or among another other arguments as shown above.

**Expected behavior**
It should get rendered.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version: 11.1
 
 **Additional context**
When I dump `$request->toJson()`, here I can see the `null` in the payload content.
https://github.com/spatie/ray/blob/69699770736030fec47ef685ab0c3a2eba8013b8/src/Client.php#L28-L29
I'm assuming this is the last possible step before the actual Mac app handles it.
Unless I'm missing something, the code for the desktop app is (understandably) not available, otherwise I'd try attempting a fix.

**Comment by freekmurze** [comment]
This is fixed in Ray 1.3.3, which will be released in a couple of minutes

**Comment by jasonvarga** [comment]
Awesome, thank you!
